.github/workflows/pr.yml

```diff
@@ -33,14 +33,17 @@ jobs:
           # These are regex patterns auto-wrapped in `^ $`.
           scopes: |
             dapi
+            platform
             drive
+            drive-abci
             dpp
             dashmate
             contract
             sdk
             wallet-lib
             test-suite
             bench-suite
+            ci
```

Comment:
There is a commit type "ci" already included with conventional commits. e.g "ci: made life better"

Reply:
I will remove it